src/components/utils/DraggableComponent.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+import Draggable from "react-draggable";
+import "../../styles/components/utils/DraggableComponent.css";
+
+function DraggableComponent({ children }) {
+  const [x, setX] = useState(0);
+  const [y, setY] = useState(0);
+
+  return (
+    <Draggable
+      defaultClassName="draggable-component"
+      defaultClassNameDragging="dragging-component"
+      defaultClassNameDragged="dragged-component"
+      bounds={{ left: 0, top: 0 }}
+      position={{ x: x, y: y }}
+      onDrag={(e, data) => {
```

Comment:
What is this?

Reply:
We have bound the draggable component, so that we don't need to implement this function inside the DraggableComponent.